### ISSUE TITLE
Crash self.stringsDataSource = StringsLocalizationDataSource(strings: strings) in LocalizationProvider

### ISSUE THREAD

**Issue by Radzievskyi** [crowdin/mobile-sdk-ios#199]
**Describe the bug**
We received crash report from Crowdin SDK file

**To Reproduce**
1. In our application on launch app, from AppDelegate.swift in:

`func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool`

we call:

```
 class func initCrowdin() {
        let crowdinProviderConfig = CrowdinProviderConfig(hashString: WXCrowdinManager.distributionHash,
                                                          sourceLanguage: WXLocalization.detectLanguage())

        let crowdinSDKConfig = CrowdinSDKConfig.config().with(crowdinProviderConfig: crowdinProviderConfig)
        
        CrowdinSDK.startWithConfig(crowdinSDKConfig, completion: {})

        CrowdinSDK.currentLocalization = WXLocalization.detectLanguage();
    }
```

1*. Also `initCrowdin() ` we call on AppleWatch in class:
`class ExtensionDelegate: NSObject, WKExtensionDelegate`
on 

`func applicationDidBecomeActive() {`

2. In Xcode and AppstoreConnect we received information about crash. Detail information is shown on screenshots.

**Expected behavior**
No crash happened

**Screenshots**
<img width="851" alt="Screen Shot 2022-08-01 at 12 52 42" src="https://user-images.githubusercontent.com/9761780/182123948-b0f770f2-c3b8-4a1f-a09b-17553cb37923.png">

<img width="1350" alt="Screen Shot 2022-08-01 at 12 55 33" src="https://user-images.githubusercontent.com/9761780/182124383-e1a29cf0-0e9e-4a4a-9465-4bcc1deb1c91.png">

<img width="1341" alt="Screen Shot 2022-08-01 at 12 55 44" src="https://user-images.githubusercontent.com/9761780/182124389-02fdbf95-a2fb-4712-b4fa-39edb93b3a7c.png">

<img width="1346" alt="Screen Shot 2022-08-01 at 12 55 51" src="https://user-images.githubusercontent.com/9761780/182124391-e52d35be-1b4f-4ed5-a11b-6dc4f87feb8d.png">

<img width="1344" alt="Screen Shot 2022-08-01 at 12 55 59" src="https://user-images.githubusercontent.com/9761780/182124392-0fc59b79-92d2-4871-9c0a-14b7a193388e.png">

<img width="1347" alt="Screen Shot 2022-08-01 at 12 56 06" src="https://user-images.githubusercontent.com/9761780/182124396-64e444ba-20d8-41e3-a406-0a8d9a23f1a6.png">

**Smartphone (please complete the following information):**
 - Device: iPhone 13 Pro Max
 - OS: iOS 15.6 (19G71)

**Comment by andrii-bodnar** [comment]
Hi @Radzievskyi!

Is there any way to reproduce this issue on a sample iOS project?

**Comment by Radzievskyi** [comment]
> Hi @Radzievskyi!
> 
> Is there any way to reproduce this issue on a sample iOS project?

Hello @andrii-bodnar 

Sorry, I got this crash log report from our TestFlight on AppstoreConnect. I can't exactly reproduce it on my device. So I can't build representative a sample iOS project.

But I checked log what I show previous more detail, and I found, when crash happened in Thread 2. Is have another Thread 9, what look similar to 2:

<img width="1326" alt="Screen Shot 2022-08-01 at 16 34 48" src="https://user-images.githubusercontent.com/9761780/182164872-517b0db2-ccaf-4827-8371-1c77545478de.png">
